package.json

```diff
@@ -30,10 +30,7 @@
     "test_network_down": "docker-compose -f ./__tests__/integration/docker-compose.yml down && rm ./tmp/wallets.json",
     "lint": "eslint .",
     "lint_fix": "eslint . --fix",
-    "generate_words": "babel-node scripts/generate_words.js",
     "generate_wallets": "babel-node scripts/generate_wallets.js",
```

Comment:
Why didn't you remove this?

Reply:
Generate wallets may be a script but it is intended to generate pre-calculated wallets for the integration tests.
The only commands moved to the Makefile were the ones intended to be called by the user.